github-action/qovery/application.go

```diff
@@ -0,0 +1,22 @@
+package qovery
+
+import (
+	"errors"
+	"fmt"
+	"github-action/pkg"
+)
+
+func GetApplicationIdByName(qoveryAPIClient pkg.QoveryAPIClient, environmentId string, name string) (string, error) {
+	applications, err := qoveryAPIClient.ListApplications(environmentId)
+	if err != nil {
+		return "", err
+	}
+
+	for _, app := range applications {
+		if app.Name == name {
+			return app.ID, nil
+		}
+	}
+
+	return "", errors.New(fmt.Sprintf("Can't find application with name %v! (it's case sensitive)", name))
```

Comment:
OPT: Won't case sensitivity be not so good here? IMO accepting names should ease integration, but case sensitivity might fade a bit this, not big issue though.

Reply:
I think our API is case sensitive so if a resource is named "Hello" and "hello" it's 2 different ones.

Reply:
👍